t/lib/t/MusicBrainz/Server/Validation.pm

```diff
@@ -278,4 +279,29 @@ test 'Test normalise_strings' => sub {
 
 };
 
+test 'Test is_valid_edit_note' => sub {
+    ok(!is_valid_edit_note(''), 'Empty edit note is invalid');
+    ok(is_valid_edit_note('This is a note!'), 'Standard edit note is valid');
+    ok(
+        !is_valid_edit_note('a'),
+        'Note made of just one ASCII character is invalid',
+    );
+    ok(
+        is_valid_edit_note('‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏漢'),
+        'Note made of just one kanji character is valid',
+    );
+    ok(
+        !is_valid_edit_note("  \t"),
+        'Note made of spaces and tabs is invalid',
+    );
+    ok(
+        !is_valid_edit_note('‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏ ‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏‍͏'),
```

Comment:
Can we use `\x{NNNN}` escape sequences here so it's visible which characters are in the string?

Reply:
We can't, because Critic disagrees, but we can use `\N{}` named sequences which is a lot better.